clientjs/rxhtml.js

```diff
@@ -3064,6 +3064,41 @@ var RxHTML = (function () {
     }
   };
 
+  transforms['dateFmt'] = function(format, d) {
+    if (typeof (d) == "string") {
+      // only transform strings
+      if (d == "1-01-01") return ""
+
+      // if datetime - strip out everything after the T
+      var s = d;
+      var k = s.indexOf('T');
+      if (k >= 0) {
+        s = s.substring(0, k);
+      }
+
+      let [year, month, day] = s.split("-");
+      let p = format.toLowerCase().split("/");
+      const yIndex = p.findIndex(x => x.includes("y"));
+      const mIndex = p.findIndex(x => x.includes("m"));
+      const dIndex = p.findIndex(x => x.includes("d"));
+
+      year = p[yIndex].length == 2 ? year.substring(2,4) : year;
+      month = p[mIndex].length == 1 ? parseInt(month) : month;
+      day = p[dIndex].length == 1 ? parseInt(day) : day;
+
+      const nd = new Array(p.length);
+      nd[yIndex] = year;
+      nd[mIndex] = month;
+      nd[dIndex] = day;
+
+      return nd.join("/");
```

Comment:
this is making / the default when a better approach would be to use a generic format string and allow the injection of various things. Does JavaScript have a built-in date/time formater?  If not, then I'd bias the formatting to be inline with what Java has ( https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html ) as that will provide a blueprint for future extensions.